README.md
![Xanadu](https://secure.gravatar.com/blavatar/682914aac6283e9a208f6d5e847c8c3b?s=128&ts=1400168049) 

![DOI](https://zenodo.org/badge/4102/sinfallas/xanadu-linux.png)

##### Xanadu GNU/Linux #####

Es un distribución Rolling Release basada en Debian SID garantizando un sistema innovador y 100% compatible con los paquetes de Debian o derivados, desarrollada para ser rápida, ligera y a la vez útil pensando en las necesidades del usuario y ofreciendo herramientas para el usuario avanzado.


La imagen .ISO y sus configuraciones para crearla se encuentran disponibles bajo licencia GPL v2 (GNU GENERAL PUBLIC LICENSE, Version 2, June 1991) y pueden ser descargadas desde:


##### DESCARGA XANADU #####

https://xanadulinux.wordpress.com/descarga/


##### Rápida y Ligera #####

Incorpora el entorno gráfico LXDE y el gestor de ventanas Openbox, lo que le permite mantener un consumo reducido cediendo el uso de los recursos a otras herramientas y programas. Haciendo que el entorno de escritorio sea rápido, ligero y eficiente.

##### Útil #####

Contiene herramientas para análisis forense como Wireshark, un analizador de protocolos de red, y ClamTK, aplicación que permite detectar virus en sistemas Windows. Además contiene aplicaciones como Gigolo el cual provee una interfaz sencilla para conectarse a carpetas compartidas en red y Filezilla como cliente FTP.

Además incluye algunas herramientas para rescate de sistemas tal como Testdisk, Foremost, Magicrescue y utilidades de análisis forense, tal como Drbl, Autopsy, Sleuthkit.

##### Segura #####

Para el equipo de Xanadu es importante la seguridad por lo que se incluyen programas como Shorewall, Fail2ban y Psad, que pueden activarse desde el post-instalador con sus configuraciones por defecto. Así como también cuenta con sistemas de detección de intrusos como Aide y para detección de rootkits incluye Rkhunter.

##### Navegación Anónima #####

También incorpora i2p y TOR para la navegación anónima complementándolos con Polipo para hacer cache y Privoxy que se encarga de filtrar contenido, adicionalmente durante la ejecución de tor Dnsmasq se encarga de torificar las peticiones DNS para asegurar que su navegación se mantenga anónima. Mediante estas herramientas es fácil replicar el modelo cliente-servidor utilizado por la distribución Whonix pero con un consumo reducido de recursos.

##### Ofimática y Multimedia #####

En este apartado, Xanadu incluye la ultima versión de la suite Libreoffice, VLC como reproductor de vídeo y Lxmusic como reproductor de audio, sin dejar atrás a brasero como programa de grabación de discos.

##### Mensajería y adicionales #####

En este apartado se incluye la aplicación Pidgin para mensajería y como gestor de archivos tenemos a Thunar y Pcmanfm.


##### Requerimientos mínimos #####

	* 512 Mb RAM (1 Gb RAM recomendado)
	* CPU a 1.0 Ghz o superior (1.5 Ghz recomendado)
	* Tarjeta de video a 800x600, 16 Mb vídeo RAM
	* BIOS con capacidad de arranque por USB o CDROM
	* 5 Gb de capacidad de disco para instalar

El nombre Xanadu se ha considerado como sinónimo de opulencia y misterio, sobre todo gracias al poema de Samuel Taylor Coleridge; también fue la capital de verano del imperio Mongol de Kublai Jan y además fue el nombre seleccionado por Theodore Holm Nelson para su idea de: 

<blockquote>
  <p>“concebir un documento global y único (docuverse), que cubra todo lo escrito en el mundo, mediante una gran cantidad de ordenadores interconectados, que contenga todo el Conocimiento existente o, mejor dicho, información en forma de hipertexto.”</p>
</blockquote>

Si quiere hacer una donación, puede realizarla mediante PayPal a: sinfallas@gmail.com

Paquetes incluidos:

https://xanadulinux.wordpress.com/paquetes-incluidos/


changelog
commit 74e66fc4cab1a48906dcba2a09ff02a16cabda3b (HEAD, origin/master, origin/HEAD, master)
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 30 22:51:58 2014 -0430

    * se sacan de la lista negra las librerias necesarias para x11vnc

commit 364ea3f0ae1e216ea2d75705d34a44fa368cc3d1
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 30 17:59:16 2014 -0430

    * changelog

commit 1e9e1b0b30b38469f6284610eaa99b904c6ba6b3
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 30 17:58:41 2014 -0430

    * se reactiva ctrl+alt+backspace par reiniciar las X

commit 037381d3aa5300454d2401b1b1cefc3311d7f250
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 30 17:26:24 2014 -0430

    * changelog

commit 4eb548c038b12a9f575ae15ab015d034974410b4
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 30 17:25:51 2014 -0430

    * se elimina manpages-dev

commit 35c0c6637301729136a05e25f1a23db96bd14094
Author: sinfallas <sinfallas@gmail.com>
Date:   Sun Oct 26 19:14:02 2014 -0430

    * changelog

commit d0a5fe98ef2d96bbc7424c51b9077b6150a85e58
Author: sinfallas <sinfallas@gmail.com>
Date:   Sun Oct 26 19:12:07 2014 -0430

    * bugfix

commit 3883944bbd45f88247b7549a8a200c0c5915327c
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 23 21:31:30 2014 -0430

    * changelog

commit 7663b9907b32561089fad154748f1c30a5ecf15c
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 23 21:31:08 2014 -0430

    * se empaqueta el theme xanadu

commit 9460025c3e0d5fb38df402f956068814d9e00de2
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 23 21:13:46 2014 -0430

    * changelog

commit c0717b0c7f365c27ca4f6103edfb84c80b688a7c
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 23 21:12:48 2014 -0430

    * se mudan las configuraciones de pcmanfm, lxterminal y el tema a sus paquetes

commit 3c3a33bfa998fccea59d0b771907d61fd5a8800a
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 23 10:38:20 2014 -0430

    * changelog

commit 942d8797ec5a0dcc5dd60f08492a88ce4d10fe49
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 23 10:37:48 2014 -0430

    * se pasa la configuracion de memlockd a su paquete

commit 952098d0a6c3e894b0f28e84d7c94b049923b4dd
Author: sinfallas <sinfallas@gmail.com>
Date:   Wed Oct 22 22:00:16 2014 -0430

    * changelog

commit d1ce7783f46001571c7449c5fb53781005febf1d
Author: sinfallas <sinfallas@gmail.com>
Date:   Wed Oct 22 21:59:43 2014 -0430

    * bugfix

commit 7fdce0ad2794870f8d66c88468b248a2c30ac1fd
Author: sinfallas <sinfallas@gmail.com>
Date:   Wed Oct 22 15:07:46 2014 -0430

    * changelog

commit a179a83ae5ab2fbb8d71aab14075aa129107e709
Author: sinfallas <sinfallas@gmail.com>
Date:   Wed Oct 22 15:06:52 2014 -0430

    * se trasladan las configuraciones de tint2 y syslinux a sus paquetes

commit 454ea612ff62da94af2496ff6d9d7803378131ac
Author: sinfallas <sinfallas@gmail.com>
Date:   Mon Oct 20 18:09:21 2014 -0430

    * changelog

commit d0aed928ae633baef10e8d2e599ba8166f8298b4
Author: sinfallas <sinfallas@gmail.com>
Date:   Mon Oct 20 18:08:55 2014 -0430

    * se agregan los logos a su paquete

commit 9cbbe4e38ddc7f11e3adbe5498a72c6c35ad34af
Author: sinfallas <sinfallas@gmail.com>
Date:   Mon Oct 20 17:47:32 2014 -0430

    * changelog

commit 924368e3338c493beeb6cfd5e6967c365377c720
Author: sinfallas <sinfallas@gmail.com>
Date:   Mon Oct 20 17:46:59 2014 -0430

    * se trasladan los wallpapers a su paquete

commit 314a5f0278eadb66cd77e71af75a1346b57e2e05
Author: sinfallas <sinfallas@gmail.com>
Date:   Mon Oct 20 17:34:11 2014 -0430

    * changelog

commit 55e860457167b81c67bca9a19c0dffd3f0603e06
Author: sinfallas <sinfallas@gmail.com>
Date:   Mon Oct 20 17:32:45 2014 -0430

    * se eliminan las configuraciones de aide, compton, dnsmasq, iceweasel, psad, shorewall, ssh, synaptic y se trasladan a su respectivo paquete

commit 0894979c0ac9e55d3921798494905cc1a1e520b8
Author: sinfallas <sinfallas@gmail.com>
Date:   Sat Oct 18 22:27:55 2014 -0430

    * changelog

commit 1b57eb04ee337a3c60b4b904cbec61ca8c9beed0
Author: sinfallas <sinfallas@gmail.com>
Date:   Sat Oct 18 22:27:26 2014 -0430

    * se traslada el archivo a su respectivo paquete

commit 65221e67d9bdae4f47e19009b53f1e67716cf446
Author: sinfallas <sinfallas@gmail.com>
Date:   Sat Oct 18 20:53:09 2014 -0430

    * changelog

commit 555905aa1563055b9694d6953557ecb0d6be64a8
Author: sinfallas <sinfallas@gmail.com>
Date:   Sat Oct 18 20:52:31 2014 -0430

    * se crea la copia del archivo de configuracion de dnsmasq

commit 80a3701cb821e75f8895753fe5f6f3b175b274ca
Author: sinfallas <sinfallas@gmail.com>
Date:   Sat Oct 18 19:42:03 2014 -0430

    * changelog

commit ea98e19fa0737374beb502194e1e36cb39345c28
Author: sinfallas <sinfallas@gmail.com>
Date:   Sat Oct 18 19:41:36 2014 -0430

    * se corrige la funcion de limpiar logs

commit 892f63f865798d2b211c7886477b1ca1ff379f00
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 16 12:29:08 2014 -0430

    * changelog

commit d1d3c4abf6d557016237c843a019d483b5961036
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 16 12:27:38 2014 -0430

    * se trasladan las configuraciones de tor a su correspondiente paquete

commit 348894990d6242ed172f4074eef2a088c256808b
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 16 12:08:37 2014 -0430

    * changelog

commit c5715211539b01db4d5fd797a720279dc106a2cc
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 16 12:07:50 2014 -0430

    * se traslada enlace simbolico de torrc al hooks

commit d245eedb8e9da1aa7284b5b0f19ee922915a59f3
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 16 12:03:32 2014 -0430

    * changelog

commit bd32cdc824b4fb618f49d7aaddf6cb3b58191a08
Author: sinfallas <sinfallas@gmail.com>
Date:   Thu Oct 16 12:02:39 2014 -0430

    * se traslada el enlace simbolico hasta hooks

commit 8340706e9c3100be98e37400965bfd7c55bf594a
Author: Jesus Palencia <sinfallas@users.noreply.github.com>
Date:   Mon Oct 13 09:24:20 2014 -0430


config/includes.chroot/etc/iceweasel/pref/iceweasel.js
// This is the Debian specific preferences file for Iceweasel
// You can make any change in here, it is the purpose of this file.
// You can, with this file and all files present in the
// /etc/iceweasel/pref directory, override any preference that is
// present in /usr/lib/iceweasel/defaults/preferences directory.
// While your changes will be kept on upgrade if you modify files in
// /etc/iceweasel/pref, please note that they won't be kept if you
// do make your changes in /usr/lib/iceweasel/defaults/preferences.
//
// Note that lockPref is allowed in these preferences files if you
// don't want users to be able to override some preferences.

pref("extensions.update.enabled", true);

// Use LANG environment variable to choose locale
pref("intl.locale.matchOS", true);

// Disable default browser checking.
pref("browser.shell.checkDefaultBrowser", false);
pref("extensions.autoDisableScopes", 0);
pref("network.http.pipelining", true);
pref("network.http.pipelining.aggressive", true);
pref("network.http.pipelining.maxrequests", 12);
pref("network.http.pipelining.ssl", true);
pref("network.http.proxy.pipelining", true);
pref("security.ssl.enable_false_start", true);
pref("network.http.keep-alive.timeout", 20);
pref("network.http.connection-retry-timeout", 0);
pref("network.http.max-persistent-connections-per-proxy", 256);
pref("network.http.max-connections-per-server",20);
pref("network.http.pipelining.reschedule-timeout", 15000);
pref("network.http.pipelining.read-timeout", 60000);
pref("network.http.pipelining.max-optimistic-requests", 3);
pref("browser.tabs.warnOnClose", false);
pref("privacy.donottrackheader.enabled", true);
pref("privacy.donottrackheader.value";1);
pref("browser.cache.disk_cache_ssl", true);
pref("browser.cache.memory.capacity", 65536);
pref("browser.display.show_image_placeholders", false);
pref("browser.safebrowsing.appRepURL", "https://sb-ssl.google.com/safebrowsing/clientreport/download");
pref("browser.xul.error_pages.enabled", true); 
pref("config.trim_on_minimize", true);
pref("content.interrupt.parsing", true);
pref("content.maxtextrun", 8191);
pref("content.max.tokenizing.time", 300000);
pref("content.notify.ontimer", true);
pref("content.notify.backoffcount", 5);
pref("content.notify.interval", 100000);
pref("general.smoothScroll", true);
pref("content.switch.threshold", 100000);
pref("gfx.font_rendering.opentype_svg.enabled", true);
pref("gfx.font_rendering.wordcache.charlimit", true);
pref("gfx.font_rendering.wordcache.maxentries", 1000);
pref("gfx.font_rendering.directwrite.enabled", true);
pref("ui.submenuDelay", 0);
pref("nglayout.initialpaint.delay", 100);
pref("security.default_personal_cert","Ask Every Time");
pref("plugin.expose_full_path", true);
pref("network.dns.disableIPv6", true);
pref("network.dnsCacheEntries", 200);
pref("network.dnsCacheExpiration", 3600); 
pref("network.cookie.thirdparty.sessionOnly",false);

config/includes.chroot/usr/lib/refractainstaller/post-install/cleanup-install.sh
#!/usr/bin/env bash
# cleanup-install.sh
rm -f /target/etc/apt/preferences.d/exclude.pref
rm -f /target/etc/apt/preferences		
rm -f /target/etc/inittab
rm -f /target/swapfile
sed -i 's_/swapfile_#/swapfile_g' /target/etc/fstab
sed -i 's_proc_#proc_g' /target/etc/fstab
sed -i 's_miswap_#miswap_g' /target/etc/rc.local
sed -i 's_swapon -f_#swapon -f_g' /target/etc/rc.local
sed -i 's_autologin-user=user_#autologin-user=user_g' /target/etc/lightdm/lightdm.conf
sed -i 's_autologin-user-timeout=0_#autologin-user-timeout=0_g' /target/etc/lightdm/lightdm.conf
sed -i 's_#greeter-hide-users=false_greeter-hide-users=false_g' /target/etc/lightdm/lightdm.conf
sed -i 's_#session-cleanup-script=_session-cleanup-script=/usr/bin/fin_g' /target/etc/lightdm/lightdm.conf
sed -i 's_show-indicators=~language;~session;~power_show-indicators=~session;~power_g' /target/etc/lightdm/lightdm-gtk-greeter.conf
sed -i 's_#show-clock=_show-clock=true_g' /target/etc/lightdm/lightdm-gtk-greeter.conf
sed -i 's_#clock-format=_clock-format=%a, %d %b %I:%M_g' /target/etc/lightdm/lightdm-gtk-greeter.conf
sed -i 's_umask 022_umask 027_g' /target/etc/init.d/rc
sed -i 's_umask 022_umask 027_g' /target/etc/login.defs
sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT="quiet"/GRUB_CMDLINE_LINUX_DEFAULT="swapaccount=1 cgroup_enable=memory disable=1 disable_ipv6=1 security=apparmor panic=10 quiet splash"/g' /target/etc/default/grub
sed -i 's_/bin/bash_/usr/bin/fish_g' /target/etc/passwd
echo "default-user-image=/usr/share/images/desktop-base/logo.png" >> /target/etc/lightdm/lightdm-gtk-greeter.conf
echo "user ALL=(ALL) ALL" > /target/etc/sudoers.d/live
echo "umask 027" >> /target/etc/profile
echo "auth required pam_succeed_if.so user != root quiet" >> /target/etc/pam.d/lightdm
echo "proc /proc proc defaults,hidepid=2 0 0" >> /target/etc/fstab
echo "tmpfs /tmp tmpfs noatime,nosuid,noexec,nodev,rw 0 0" >> /target/etc/fstab
echo "tmpfs /run tmpfs rw,nosuid,noexec,nodev,noatime,mode=755 0 0" >> /target/etc/fstab
echo "tmpfs /var/cache/samba tmpfs noatime,nodev 0 0" >> /target/etc/fstab
echo "tmpfs /var/spool tmpfs noatime,nodev 0 0" >> /target/etc/fstab
echo "tmpfs /var/tmp tmpfs noatime,nosuid,noexec,nodev,rw 0 0" >> /target/etc/fstab
if [ -n "$(lspci | grep -E 'VGA|Display' | head -n1 | cut -d ':' -f3 | grep -F 'Intel')" ]; then
	echo 'Section "Device"' > /target/etc/X11/xorg.conf
	echo ' Identifier "intel"' >> /target/etc/X11/xorg.conf
	echo ' Driver "intel"' >> /target/etc/X11/xorg.conf
	echo ' BusID  "PCI:0:2:0"' >> /target/etc/X11/xorg.conf
	echo ' Option "AccelMethod" "SNA"' >> /target/etc/X11/xorg.conf
	echo ' Option "SwapbuffersWait" "false"' >> /target/etc/X11/xorg.conf
	echo ' Option "Tiling" "true"' >> /target/etc/X11/xorg.conf
	echo ' Option "BackingStore" "True"' >> /target/etc/X11/xorg.conf
	echo ' Option "XvMC" "on"' >> /target/etc/X11/xorg.conf
	echo ' Option "TripleBuffer" "true"' >> /target/etc/X11/xorg.conf
	echo ' Option "DRI" "true"' >> /target/etc/X11/xorg.conf
	echo 'EndSection' >> /target/etc/X11/xorg.conf
fi
chmod 600 /target/etc/sudoers
exit 0


instalar.sh
#!/bin/bash
#elaborado por sinfallas
if [[ $USER != root ]]; then
echo "Error: Debe tener privilegios de ROOT"
exit 1
fi
if [[ -f /etc/xanadu/version ]]; then
	apt-get update
	apt-get -y dist-upgrade
else
	echo "deb http://ftp.debian.org/debian testing main contrib non-free" > /etc/apt/sources.list
	echo "deb http://ftp.debian.org/debian/ jessie-updates main contrib non-free" >> /etc/apt/sources.list
	echo "deb http://security.debian.org/ jessie/updates main contrib non-free" >> /etc/apt/sources.list
	apt-get update
fi
apt-get -y install live-build live-boot live-config squid3 git
sed -i 's_tmpfs /var/spool_#tmpfs /var/spool_g' /etc/fstab
mkdir -p /var/spool/squid3
chown -hR root /var/spool/squid3
chmod -R 0666 /var/spool/squid3
service squid3 stop
echo "http_port 3128" > /etc/squid3/squid.conf
echo "icp_port 0" >> /etc/squid3/squid.conf
echo "htcp_port 0" >> /etc/squid3/squid.conf
echo "snmp_port 0" >> /etc/squid3/squid.conf
echo "coredump_dir /var/spool/squid3" >> /etc/squid3/squid.conf
echo "access_log /var/log/squid3/access.log" >> /etc/squid3/squid.conf
echo "cache_log /var/log/squid3/cache.log" >> /etc/squid3/squid.conf
echo "cache_mem 256 MB" >> /etc/squid3/squid.conf
echo "maximum_object_size_in_memory 1024 KB" >> /etc/squid3/squid.conf
echo "maximum_object_size 50 MB" >> /etc/squid3/squid.conf
echo "cache_replacement_policy heap LFUDA" >> /etc/squid3/squid.conf
echo "memory_replacement_policy heap LFUDA" >> /etc/squid3/squid.conf
echo "cache_dir aufs /var/spool/squid3 8192 128 128" >> /etc/squid3/squid.conf
echo "pipeline_prefetch on" >> /etc/squid3/squid.conf
echo "shutdown_lifetime 2 second" >> /etc/squid3/squid.conf
echo "acl all src all" >> /etc/squid3/squid.conf
echo "acl localhost src 127.0.0.1/32" >> /etc/squid3/squid.conf
echo "acl to_localhost dst 127.0.0.0/8 0.0.0.0/32" >> /etc/squid3/squid.conf
echo "acl localnet src 192.168.1.0/24    # internal network" >> /etc/squid3/squid.conf
echo "acl SSL_ports port 443        # https" >> /etc/squid3/squid.conf
echo "acl SSL_ports port 563        # snews" >> /etc/squid3/squid.conf
echo "acl SSL_ports port 873        # rsync" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 80        # http" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 21        # ftp" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 443        # https" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 70        # gopher" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 210        # wais" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 1025-65535    # unregistered ports" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 280        # http-mgmt" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 488        # gss-http" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 591        # filemaker" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 777        # multiling http" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 631        # cups" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 873        # rsync" >> /etc/squid3/squid.conf
echo "acl Safe_ports port 901        # SWAT" >> /etc/squid3/squid.conf
echo "acl downloads urlpath_regex [-i] \.dll$ \.bin$ \.cab$ \.asx$ \.vbs$ \.bat$ \.lnk$ \.scr$ \.pif$ \.msi$ \.exe$ \.mp3$ \.wmv$ \.zip$ \.mpg$ \.torrent$ \.ppt$ \.rar$ \.avi$ \.flv$ \.pps$ \.wma$" >> /etc/squid3/squid.conf
echo "acl purge method PURGE" >> /etc/squid3/squid.conf
echo "acl CONNECT method CONNECT" >> /etc/squid3/squid.conf
echo "http_access allow purge localhost" >> /etc/squid3/squid.conf
echo "http_access deny purge" >> /etc/squid3/squid.conf
echo "http_access allow to_localhost" >> /etc/squid3/squid.conf
echo "http_access allow localnet" >> /etc/squid3/squid.conf
echo "http_access allow localhost" >> /etc/squid3/squid.conf
echo "http_access allow downloads" >> /etc/squid3/squid.conf
echo "http_access deny !Safe_ports" >> /etc/squid3/squid.conf
echo "http_access deny CONNECT !SSL_ports" >> /etc/squid3/squid.conf
echo "http_access allow all" >> /etc/squid3/squid.conf
squid3 -z
service squid3 start
echo "export http_proxy=http://127.0.0.1:3128/" >> /root/.bashrc
echo "export https_proxy=http://127.0.0.1:3128/" >> /root/.bashrc
echo "export ftp_proxy=http://127.0.0.1:3128/" >> /root/.bashrc
mkdir xanadu
cd xanadu
lb config --config git://github.com/sinfallas/xanadu-linux.git
echo "Instalación Finalizada, debe reiniciar el equipo.."
exit 0
